packages/api/src/controllers/webhook.ts

```diff
@@ -287,9 +295,45 @@ const requestsFieldsMap: FieldsMap = {
   statusCode: `webhook_response.data->'response'->>'status'`,
 };
 
+app.put("/:id/log/:requestId/resend", authorizer({}), async (req, res) => {
+  const webhook = await db.webhook.get(req.params.id);
+  const webhookResponse = await db.webhookResponse.get(req.params.requestId);
+  await checkRequest(req, webhook, webhookResponse);
+
+  await resendWebhook(webhook, webhookResponse);
+  res.status(202).end();
+});
+
+app.get("/:id/log/:requestId", authorizer({}), async (req, res) => {
+  const webhook = await db.webhook.get(req.params.id);
+  const webhookResponse = await db.webhookResponse.get(req.params.requestId);
+  await checkRequest(req, webhook, webhookResponse);
+
+  res.status(200);
+  return res.json(webhookResponse);
+});
+
+async function checkRequest(
+  req,
```

Comment:
**ts-nit:** You can change the `express` import to 
```ts
import { Request, Router } from "express";
```
and use the `Request` type here (it will also fix the typing of all the route handlers 🙏)

Reply:
I found that I needed to use `import Router from "express/lib/router";` otherwise I get errors on the queryparams like limit and cursor:
```TS2322: Type 'string | ParsedQs | string[] | ParsedQs[]' is not assignable to type 'string | number'.   Type 'ParsedQs' is not assignable to type 'string | number'.```

Reply:
Yeah those are actual errors we would get in production! I think if a query-string is specified multiple times it will come as an array to the API handlers. To fix these, I generally convert the query-string to all strings, with `toStringValues`: e.g. https://github.com/livepeer/livepeer-com/blob/80f3aa14daed6a03eba294b490544e0725796472/packages/api/src/controllers/stream.ts#L1010